models/classes/container/QtiTestDeliveryContainer.php

```diff
@@ -65,6 +66,15 @@ public function getExecutionContainer(DeliveryExecution $execution)
         $container->setData('serviceCallId', $execution->getIdentifier());
         $container->setData('deliveryExecution', $execution->getIdentifier());
         $container->setData('deliveryServerConfig', []);
+
+        //setting tenantId/Name if it exists (and we are in multi-tenant context)
+        //as for now it is needed for textHelp caching as a bookId
+        $tenantService = $this->getServiceLocator()->get(ContainerService::SERVICE_ID);
+
+        $tenantName = method_exists($tenantService, 'getTenantName') ? $tenantService->getTenantName() : false;
+
+        $container->setData('tenantName', $tenantName);
```

Comment:
you cannot pass this information like that, this  service is aimed at providing very generic data about the the delivery.
you may want to look into examples of the best practice about configuring a test plugin, e.g. check the updater of taoQtiTest or taoAct, you got example like
```
$config = $extension->getConfig('testRunner');

           

            $config['plugins']['magnifier'] = [
                'zoomMin' => 2,
                'zoomMax' => 8,
                'zoomStep' => .5
            ];

            $extension->setConfig('testRunner', $config);
```
Call it in the extension you want this specific information to be added.